docs/DEVELOPING.md

```diff
@@ -441,6 +441,97 @@ $ make ftest NAME=sharepoint
 
 ℹ️ The connector uses the Elastic [ingest attachment processor](https://www.elastic.co/guide/en/enterprise-search/current/ingest-pipelines.html) plugin for extracting file contents. The ingest attachment processor extracts files by using the Apache text extraction library Tika. Supported file types eligible for extraction can be found as `TIKA_SUPPORTED_FILETYPES` in [utils.py](../connectors/utils.py) file.
 
+## ServiceNow Connector
+
+The [Elastic ServiceNow connector](../connectors/sources/servicenow.py) is provided in the Elastic connectors python framework and can be used as a [connector client](https://www.elastic.co/guide/en/enterprise-search/current/build-connector.html).
+
+### Availability and prerequisites
```

Comment:
It seems like there should be some information on what user and role is required to query the services.